app/components/Header/index.js

```diff
@@ -56,27 +56,29 @@ class Header extends Component {
     this.setState({ showUnsavedWarning: true });
   }
 
+  /**
+   * Special case since we need to render JSX inside the ErrorMessage component
+   */
   _renderUnsavedWarning() {
     if (!this.state.showUnsavedWarning) {
       return '';
     }
 
-    function discardChanges() {
+    const discardChanges = function discardChanges() {
```

Comment:
IE: move it outside of the render function(s) and bind it in the constructor like everything else